src/rust/rust-proto-new/src/lib.rs

```diff
@@ -151,6 +160,11 @@ pub mod graplinc {
                 mod v1beta1_client;
                 mod v1beta1_server;
             }
+
+            #[cfg(feature = "uid-allocator")]
+            pub mod uid_allocator {
+                pub mod v1beta1;
+            }
```

Comment:
it's weird to me that you cfg guard the native code but not the protobuf generated stuff on 90

Reply:
I should do that too